x-pack/plugins/case/server/client/cases/create.test.ts

```diff
@@ -276,14 +277,16 @@ describe('create', () => {
         caseSavedObject: mockCases,
       });
       const caseClient = await createCaseClientWithMockSavedObjectsClient({ savedObjectsClient });
-      caseClient.client
-        // @ts-expect-error
-        .create({ theCase: postCase })
-        .catch((e) => {
-          expect(e).not.toBeNull();
-          expect(e.isBoom).toBe(true);
-          expect(e.output.statusCode).toBe(400);
-        });
+      return (
```

Comment:
Adding `return` to tests so that the test will wait correctly.

See: https://jestjs.io/docs/en/asynchronous#promises